code/CMSEditLinkExtension.php

```diff
@@ -71,6 +71,7 @@ public function getCMSEditLinkForManagedDataObject(DataObject $obj, string $reci
 
     /**
      * Get a link to edit this DataObject in the CMS.
+     * @deprecated 2.4.0 Will be replaced with SilverStripe\ORM\DataObject::getCMSEditLink() and updateCMSEditLink()
```

Comment:
This should have been done along-side https://github.com/silverstripe/silverstripe-admin/pull/1809